src/components/homepage/homepage.component.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+import styled from 'styled-components'
+import {Route, Switch, Link} from 'react-router-dom'
+
+
+import PizzaForm from '../../components/pizza-form/pizza-form.component'
+
+
+const NavBar = styled.div `
+width: 50%;
+border-bottom: 2px solid black;
+
+
+`
+
+
+const Homepage = () => {
+
+  return (
+    <div>
```

Comment:
Good use of components, the only thing I would improve is your indentation, it's difficult to tell where child elements/components are